python/tvm/relay/op/strategy/generic.py

```diff
@@ -1793,6 +1802,18 @@ def cumsum_strategy(attrs, inputs, out_type, target):
     return strategy
 
 
+@override_native_generic_func("concat_strategy")
+def concatenate_strategy(attrs, inputs, out_type, target):
+    """concatenate generic strategy"""
+    strategy = _op.OpStrategy()
+    strategy.add_implementation(
+        wrap_compute_concat(topi.concatenate),
+        wrap_topi_schedule(topi.generic.schedule_extern),
```

Comment:
I think this shouldn't be `schedule_extern`.

Reply:
Yes, it should be `schedule_injective`. Things in `generic.py` are only used by cpu targets (cuda has its own strategies for concat etc).

Reply:
I tried, the result is here: https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-11341/26/pipeline/456

25-th build was ok.

Reply:
yeah, I think you also need to define `concatenate_strategy` in `strategy/cuda.py`, and use `topi.cuda.schedule_injective` there. That should allow using `schedule_injective` here.

In general, if you add a new strategy in `generic.py`, you should also update gpu strategy in `strategy/cuda.py` as well.

Reply:
Done